src/unix/notbsd/linux/mod.rs

```diff
@@ -83,9 +83,11 @@ s! {
     }
 
     pub struct pthread_mutexattr_t {
-        #[cfg(target_arch = "x86_64")]
+        #[cfg(any(target_arch = "x86_64", target_arch = "powerpc64",
+                  target_arch = "powerpc64le"))]
```

Comment:
This hit an aarch64 fail, looks like it requires 8 byte alignment: https://travis-ci.org/rust-lang-nursery/libc/jobs/101784088

I'll back that change out